src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/terminal.utils.tsx

```diff
@@ -329,6 +330,30 @@ const manifestEditButtons = ({
     )
 }
 
+const downloadFileFolderButton = (elementData) => {
```

Comment:
Can extend type from terminalStripTypeData through Parameters and add return type and ideally should be a component